src/main/kotlin/com/evalify/evalifybackend/notification/repository/NotificationRepository.kt

```
@@ -0,0 +1,11 @@
+package com.evalify.evalifybackend.notification.repository
+
+import com.evalify.evalifybackend.notification.domain.Notification
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.rest.core.annotation.RepositoryRestResource
+import java.util.UUID
+
+
+@RepositoryRestResource(path = "notification")
+interface NotificationsRepository : JpaRepository<Notification, UUID>{
+}
```

detekt (1.23.8)

[warning] The class or object NotificationsRepository is empty.

(detekt.empty-blocks.EmptyClassBlock)